app.js

```diff
@@ -57,9 +57,6 @@ app.use(compileSass({
     logToConsole: false // If true, will log to console.error on errors
 }));
 app.use(express.static(path.join(__dirname, 'public')));
-app.use('/bower_components', express.static(path.join(__dirname, '/bower_components')));
-
-app.use('/bower_components', express.static(path.join(__dirname, '/bower_components')));
```

Comment:
note:
app.use('/public/node_modules', express.static(path.join(__dirname, '/node_modules'))); 
is not required, since node_modules is already a part of public (which is signed as static - line 59);